libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/validator/VerifiableCredentialValidator.java

```diff
@@ -49,38 +50,50 @@ DefaultJWTClaimsVerifier<SimpleSecurityContext> createVerifier(
 
     private IClaimsVerifierFactory claimsVerifierFactory = DefaultJWTClaimsVerifier::new;
 
-    public VerifiableCredentialJwtValidator(ConfigService configService) {
+    public VerifiableCredentialValidator(ConfigService configService) {
         this.configService = configService;
     }
 
     // Constructor for tests where we need to use a fixed time.
-    public VerifiableCredentialJwtValidator(
+    public VerifiableCredentialValidator(
             ConfigService configService, IClaimsVerifierFactory claimsVerifierFactory) {
         this.configService = configService;
         this.claimsVerifierFactory = claimsVerifierFactory;
     }
 
     public List<VerifiableCredential> validate(
-            List<String> vcs, String criId, ECKey signingKey, String componentId, String userId)
+            String userId,
+            String criId,
+            List<String> vcs,
+            String vcType,
+            ECKey signingKey,
+            String componentId)
             throws VerifiableCredentialException {
         var validVcs = new ArrayList<VerifiableCredential>();
         for (String vc : vcs) {
-            validVcs.add(validate(vc, criId, signingKey, componentId, userId));
+            validVcs.add(validate(userId, criId, vc, vcType, signingKey, componentId, false));
         }
-
         return validVcs;
     }
 
     public VerifiableCredential validate(
```

Comment:
You could consider making this one private and adding a new public one `validateWithNoSubjectCheck`.

It's not necessary, but avoids having a boolean flag on the public interface (which can be hard to read).

Reply:
I think I will keep it as it is for now since it's at least currently quite satisfying having verifiable credential created from the validation in one method... Unless you think there is enough to merit the change despite this

Reply:
Sure - I mean we currently have two public methods here anyway, but I don't feel very strongly about it.